### PR TITLE
Add common misspellings of RISC-V

### PR DESCRIPTION
**Pull request by felixonmars**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -29108,6 +29108,10 @@ riminders->reminders
 riminding->reminding
 rimitives->primitives
 rininging->ringing
+rsicv->riscv
+RSICV->RISCV
+rsic-v->risc-v
+RSIC-V->RISC-V
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:29114]: So we always lowercase the LHS, but ideally the left should be what case is appropriate. It's not used currently but it's there for the future!

**Review by peternewman — CHANGES_REQUESTED** [review]
Thanks

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (felixonmars) [reply codespell_lib/data/dictionary.txt]: Hope I get it right this time :)

**Comment by luzpaz** [comment]
LGTM

**Comment by skangas** [comment]
LGTM

**Review by DimitriPapadopoulos — APPROVED** [review]

**Review by skangas — APPROVED** [review]

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**